OpenTrons/opentrons-api
# refactor(app): Style drop tip flows

Closes RQA-1939, RQA-1957, RQA-1938, RQA-1941, RQA-1940



# Overview

This PR fixes a number of items related to the drop tip wizard flows. The only new functional add is the "Go back" ability in ChooseLocation, which required moving the create maintenance run logic to BeforeBeginning (functionally, this does not impact anything).

There's a few too many pictures to include in this PR, but consult each ticket if interested. 


# Test Plan

- Open the tickets and run through the desktop and ODD drop tip flows. Compare the correct version images to what is now rendered (see convo on RQA-1941). 
- Try out the new "Go back" button in ChooseLocation and make sure it behaves appropriately.


# Changelog
- Fixed several style issues within the Drop Tip Wizard flows.

# Risk assessment
low
